elemica-suggest.coffee

```diff
@@ -249,7 +262,10 @@ elemicaSuggest 0.9.2-SNAPSHOT
               highlightNext(this)
             when key is ENTER
               selectHighlighted(this)
-            else
+            # Ignore other non-printable keys.
+            when key isnt CTRL && key isnt ALT && key isnt SHIFT &&
+                 key isnt OS_LEFT && key isnt OS_RIGHT &&
+                 key isnt TAB
```

Comment:
What do you think about putting all these in an array ie. `NON_PRINTABLE_KEYS` and then instead of this chain use the array: `when key not in NON_PRINTABLE_KEYS`?